package.json

```diff
@@ -8,12 +8,40 @@
     "build:app": "vuepress build docs",
     "build:lambda": "netlify-lambda build lambda",
     "docs:build": "vuepress build docs",
-    "postinstall": "netlify-lambda install"
+    "postinstall": "netlify-lambda install",
+    "test": "./node_modules/.bin/jest --silent"
   },
   "author": "Alan Woo (Bin Studio) <alan@bin.am>",
   "license": "MIT",
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.js$": "<rootDir>/node_modules/babel-jest"
+    },
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/docs/.vuepress/$1",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__test__/mocks/FileMock.js"
+    },
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "docs/.vuepress/theme/**/*.{js,vue}",
+      "!**/node_modules/**"
+    ]
+  },
   "devDependencies": {
     "@vuepress/plugin-active-header-links": "^1.0.0-rc.1",
+    "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-jest": "^24.9.0",
+    "babel-preset-env": "^1.7.0",
+    "jest": "^24.9.0",
+    "jest-serializer-vue": "^2.0.2",
+    "vue-jest": "^3.0.5",
+    "babel-core": "^6.26.3",
```

Comment:
Same here - any reason to use `babel-core` vs `@babel/core`? Noticing that's what's mentioned in jest docs: https://jestjs.io/docs/en/getting-started#using-babel

Reply:
Yep, there some bug by using "@babel/core" with vue/jest: https://github.com/vuejs/vue-jest/issues/160. But i added https://github.com/babel/babel-bridge to solve this.